x-pack/plugin/ml/src/main/java/org/elasticsearch/xpack/ml/inference/pytorch/process/NativePyTorchProcess.java

```diff
@@ -7,26 +7,41 @@
 
 package org.elasticsearch.xpack.ml.inference.pytorch.process;
 
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import org.elasticsearch.common.xcontent.NamedXContentRegistry;
+import org.elasticsearch.xpack.core.ml.inference.deployment.PyTorchResult;
 import org.elasticsearch.xpack.ml.process.AbstractNativeProcess;
 import org.elasticsearch.xpack.ml.process.NativeController;
 import org.elasticsearch.xpack.ml.process.ProcessPipes;
+import org.elasticsearch.xpack.ml.process.ProcessResultsParser;
 import org.elasticsearch.xpack.ml.process.writer.LengthEncodedWriter;
 
 import java.io.IOException;
 import java.io.OutputStream;
 import java.nio.charset.StandardCharsets;
 import java.nio.file.Path;
+import java.util.Arrays;
 import java.util.Base64;
+import java.util.Iterator;
 import java.util.List;
+import java.util.concurrent.atomic.AtomicLong;
 import java.util.function.Consumer;
 
 public class NativePyTorchProcess extends AbstractNativeProcess implements PyTorchProcess {
 
+    private static final Logger logger = LogManager.getLogger(NativePyTorchProcess.class);
+
     private static final String NAME = "pytorch_inference";
 
+    private static AtomicLong ms_RequestId = new AtomicLong(1);
```

Comment:
what does the `ms_` mean?

Reply:
(m)ember (s)tatic. Not sure if we're supposed to be using this convention. In all fairness, I saw we did it this way in `AutodetectControlMsgWriter.ms_FlushNumber` but perhaps that's too ancient to follow :-)

Reply:
It’s the old Prelert convention that matched the C++ standards. We took these out of the Java code in 2016 but must have missed that one.

Reply:
cool, I'll change it then. And raise a tiny PR to fix the flush one too.